cli/src/pcluster/constants.py

```diff
@@ -64,8 +64,9 @@
 MAX_NUMBER_OF_QUEUES = 10
 MAX_NUMBER_OF_COMPUTE_RESOURCES = 5
 
-# Considering the limited size of CloudFormation template and API throttling, we decided to support at most 10 EFS.
-MAX_STORAGE_COUNT = {"ebs": 5, "efs": 10, "fsx": 10, "raid": 1}
+MAX_EBS_COUNT = 5
```

Comment:
I'd use the same convention of the other volume type, by adding this info in the `MAX_NEW|EXISTING_STORAGE_COUNT`. 

This make the code more consistent and easy to change if we want to modify these limits for existing/new EBS volumes too.

Reply:
As discussed, we supports:
1 new EFS, 50 existing EFS
1 new FSx, 50 existing FSx
1 new RAID, 0 existing RAID
5 EBS (total of new and existing)

Reply:
It seems like the code here would be cleaner if we simply had EBS in the `MAX_NEW_STORAGE_COUNT` and `MAX_EXISTING_STORAGE_COUNT` dicts as described by @enrico-usai  which would make it more consistent.